triton/tut/interactive.rst

```diff
@@ -2,160 +2,190 @@
+Put more precisely, you want access to a node in the cluster
+through an interactive bash shell. 
+For this, you just need srun's ``--pty`` option coupled with the shell
+you want
+
+::
+
+  srun -p interactive --time=2:00:00 --mem=600 --pty bash 
+
+The command prompt will appear when the job starts.
+And you will have a bash shell runnnig on one of the 
+computation nodes with at least 600 Megabytes of memory,
+for a duration of 2 hours, where you can run your programs in. 
 
-    srun -p interactive --time=HH:MM:SS --mem=nnG --pty bash
+The option ``-p interactive`` requests a node in the interactive
+partition which is dedicated to interactive usage (more on this later). 
+
+.. note::
 
-Now you have a shell... do whatever you need to do. **Close the shell
-when you are done!  If you don't, the process will keep running until
-your time limit. All of this time will be counted against your usage.
-It doesn't cost money, but does mean that your priority will go down in
-the future.**  (Note that we specify the interactive partition with "-p
-interactive". More on this below.)
+  you can use ``sinfo`` to see information such as the available partitions,
+  number of nodes in each, their time limits etc. 
 
+.. warning::
+  
+  Remember to exit the shell when you are done!
+  The shell will be running if you don't and
+  it will count towards your usage. 
+  This effectively means your priority will degrade
+  in the future.
+  
 
 Interactive shell with graphics
 ===============================
 
-sinteractive is very similar to srun, however it is more clever and thus
-allows you to do X forwarding. In the background, it starts the job,
-starts a screen session on the node, then sshes to there and connects to
-the screen. You can also ssh to this node again and connect to the
+``sinteractive`` is very similar to ``srun``, but more clever and thus
+allows you to do X forwarding. It starts a screen session on the node, 
+then sshes to there and connects to the screen. 
+You can also ssh to this node again and connect to the
 process again.
 
 ::
 
-     sinteractive --time=HH:MM:SS --mem=nnG
+     sinteractive --time=1:00:00 --mem=1000
 
-**Just like with** ``srun --pty``, **remember to close the process when done.
-However, it's even harder than before. Since there is a separate screen
-session running, just closing the terminal isn't enough. Exit all
-shells in the screen session on the node (C-d or ``exit``), or cancel
-the process (see below).**
+.. warning::
 
-If you are off-campus, you might want to use https://vdi.aalto.fi as a
-virtual desktop to connect to Triton to run graphical programs.
-Otherwise, expect things to be very slow.
+  Just like with ``srun --pty bash``, remember to exit the shell.
+  Since there is a separate screen session running, just closing the terminal isn't enough. 
+  Exit all shells in the screen session on the node (C-d or ``exit``), or cancel
+  the process (see below).
```

Comment:
I think it actually does exit properly, I was just either wrong or or it was broken and I fixed it.  Want to test and if it does the right thing, remove this?
```suggestion
```

Reply:
The interactive partition is either drained or busy. checking periodically.